packages/eas-cli/src/build/utils/repository.ts

```diff
@@ -83,6 +83,35 @@ export async function commitPromptAsync(
   });
 }
 
+export async function makeProjectMetadataFileAsync(archivePath: string): Promise<{ path: string; size: number }> {
```

Comment:
I think "[project] archive listing" would be a better name than "project metadata". Unless you're thinking of making it reused for other things (ugh, another metadata…), then I would nest the listing under a key of the file, not at the top level.

Reply:
Oh, only when you commented I saw it's indeed nested under `archiveContent`, sorry!